WikiTemplate/Crypto/wikitemplate-cryptoAndroid.md

```diff
@@ -0,0 +1,70 @@
+
+## Onboarding
+
+- [ ] Verify able to create a wallet by following onboarding
+- [ ] Verify not able to screenshots during onboarding screens
+- [ ] Verify able to backup wallet seed phrase during onboarding
+- [ ] Verify able to skip backup wallet seed screen to complete onboarding
+- [ ] Verify onboarding is shown when trying to interact with Dapp without creating wallet
+
+## Panel
+
+- [ ] Verify panel is only shown in URL when a Dapp is trying to interact
+- [ ] Verify able to change default network from panel
+- [ ] Verify able to change accounts from panel
+- [ ] Verify notification dot is shown on wallet icon when an action is pending for Wallet
+
+## Ethereum
+
+- [ ] Verify Ethereum account is created as soon as onboarding is completed
+- [ ] Verify able to create an Ethereum account can be created from accounts tab
+- [ ] Verify able to create an Ethereum account can be created from accounts list screen
+- [ ] Verify able to import an Ethereum account via Private Key/JSON file
+- [ ] Verify able to submit a Buy ETH transaction
```

Comment:
Not sure how much work this will add, but recently we had a bug related to locale: https://github.com/brave/brave-browser/issues/24909
Will it possible to test transactions in US/EU locales?

Reply:
I'll add a test for locale

Reply:
There are two clauses here:

> Verify able to create an Ethereum account can be created from accounts tab

`able to create` and `can be created` - either would be fine, your choice of course